include/xtensor/xslice.hpp

```diff
@@ -119,6 +119,15 @@ namespace xt
         size_type m_step;
     };
 
+    struct xnone_tag
+    {
+    };
+
+    inline auto xnone()
+    {
+        return xnone_tag();
+    }
```

Comment:
I think we can use placeholders here, so we have a really cool syntax:
```cpp
auto v = view(exp, range(2, _, 2));
```
Although ``_`` is not a in the STL yet, it is in the MPL and it may become available in the STL in the future. So we have to be very cautious. A good approach would be to nest it in a placeholder namespace, forcing the client to explicitly use the namespace if he wants to use the placeholder syntax. This way, when the ``_`` placeholder comes in the standard, we can just replace our code by an import of the standard one.
A possible implementation:
```cpp
namespace placeholders
{
    // xtensor universal placeholder
    struct xtuph
    {
    };

    constexpr xtuph _{};
}
```